google-cloud-spanner/test/google/cloud/spanner/client/transaction_retry_test.rb

```diff
@@ -84,7 +84,7 @@
     mock.expect :create_session, session_grpc, [database_path(instance_id, database_id), options: default_options]
     mock.expect :begin_transaction, transaction_grpc, [session_grpc.name, tx_opts, options: default_options]
     mock.expect :execute_streaming_sql, results_enum, [session_grpc.name, "SELECT * FROM users", transaction: tx_selector, params: nil, param_types: nil, resume_token: nil, options: default_options]
-    # mock.expect :commit, ->{ raise GRPC::Aborted }, [session_grpc.name, mutations, transaction_id: transaction_id, single_use_transaction: nil, options: default_options]
+    # mock.expect :commit, ->{ raise GRPC::Unavailable }, [session_grpc.name, mutations, transaction_id: transaction_id, single_use_transaction: nil, options: default_options]
```

Comment:
Yep. I was overzealous in my copy/paste. The actual mock didn't change, but this commented out mock was. Fixed.